Flabbergasted-Flamingos/Bangazon-admin-app
# Cd department list

# Description
Added department list view and testing

## Number of Fixes


## Related Ticket(s)
#5 

## Problem to Solve
Added department list view.

## Proposed Changes
1. added department_list.py in views directory
2. updated urls.py with the url for departments
3. added a tests folder

## Expected Behavior
Departments should be listed at the url:
http://127.0.0.1:8000/admin_app/departments/

all python testing should pass.

## Steps to Test Solution

make sure you are in the Bangazon-Admin-App directory

Department List:
```
git fetch --all
git checkout cd_department_list
python manage.py runserver
```
1. go to url:
http://127.0.0.1:8000/admin_app/departments/

2. make sure that there is a list of all of the department names in your db.

Test:
```
python manage.py test
```
1. Make sure all of the tests pass.

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
